src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -73,218 +73,209 @@ const LINKS = {
-              <div
-                className={
-                  showMobileMenu ? styles.mobileMenu : styles.hideMobileMenu
-                }
-              >
-                <NavLink
-                  href={logoHref}
-                  LinkComponent={LinkComponent}
-                  className={styles.phoneLogo}
-                >
-                  {LogoWhite({ className: styles.logo })}
-                </NavLink>
-                <Spacer.H56 />
-                {LINKS.NAVLINKS.map((link) => (
-                  <div key={link.id} style={{ marginBottom: 24 }}>
-                    <TitleXLarge.Sans.Regular400>
-                      <NavLink
-                        href={link.href}
-                        LinkComponent={
-                          link.href !== '/login/' ? LinkComponent : null
-                        }
-                      >
-                        {link.title}
-                      </NavLink>
-                    </TitleXLarge.Sans.Regular400>
-                  </div>
-                ))}
-                <div style={{ position: 'absolute', bottom: 40 }}>
-                  <a href={LINKS.TERM.href}>
-                    <Button.Medium.WhiteOutline>
-                      Check my price
-                    </Button.Medium.WhiteOutline>
-                  </a>
-                </div>
-              </div>
+  return (
+    <div className={styles.blockNavbar}>
+      <div className={styles.navbar}>
+        <Layout.ScrollDetector>
+          <div className={styles.hamburger}>
+            <TransformingBurgerButton
+              showMobileMenu={showMobileMenu}
+              clickHandler={toggleHamburger}
+              keyPressHandler={handleHamburgerKeyPress}
+            />
+          </div>
 
-              {/* Mobile menu items, getAnEstimate only shows when scrolled */}
+          <div className={styles.phoneOnly}>
+            {/* Dark green mobile menu, shows up when hamburger is clicked */}
+            <div
+              className={
+                showMobileMenu ? styles.mobileMenu : styles.hideMobileMenu
+              }
+            >
               <NavLink
-                href={LINKS.INDEX.href}
+                href={logoHref}
                 LinkComponent={LinkComponent}
-                className={styles.phoneLogoFancy}
+                className={styles.phoneLogo}
               >
-                {FancyAnimatedLogo()}
+                {LogoWhite({ className: styles.logo })}
               </NavLink>
-              {!hideMobileCta && renderCtaButton(true)}
-              {renderSearchIcon(LINKS.NAVLINKS[3])}
+              <Spacer.H56 />
+              {LINKS.NAVLINKS.map((link) => (
+                <div key={link.id} style={{ marginBottom: 24 }}>
+                  <TitleXLarge.Sans.Regular400>
+                    <NavLink
+                      href={link.href}
+                      LinkComponent={
+                        link.href !== '/login/' ? LinkComponent : null
```

Comment:
This is duplication I think we could figure out how to remove

Reply:
same: https://github.com/getethos/ethos-design-system/pull/275/commits/bfbb5a3f0520f2f157c895f155212546bf09b3a7